modules/administration-guide/pages/extensions-for-microsoft-visual-studio-code-open-source.adoc

```diff
@@ -7,12 +7,14 @@
 [id="extensions-for-microsoft-visual-studio-code-open-source"]
 = Extensions for Microsoft Visual Studio Code - Open Source
 
-A paragraph or more of conceptual information about the embedded link:https://open-vsx.org/[Open VSX Registry] that Che admins need to understand.
+To manage extensions for link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source IDE], {prod-short} uses one of the link:https://open-vsx.org/about[Open VSX] registry instances:
```

Comment:
```suggestion
To manage extensions for link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source IDE], {prod-short} uses one of these three link:https://open-vsx.org/about[Open VSX] registry instances:
```

Reply:
There can be more than three given the unlimited third point, no?

Reply:
would you prefer "these three CATEGORIES of registry instances" ?

or just "these registries" ?

Reply:
@nickboldt, thank you, Nick. These registries work fine. I'm committing your suggestion.